template/example/README.md

```diff
@@ -1,3 +1,39 @@
 # This is a terraform module example
+The current directory contains terraform related files.
+Terraform module can contain sub modules.
 
-TODO: explain
\ No newline at end of file
+## Directory structure
+Examples:
+- minimum
+```text
+.
+├── README.md
+├── main.tf
+├── variables.tf
+├── outputs.tf
+```
+
+- with sub-modules
+```text
+├── README.md
+├── main.tf
+├── variables.tf
+├── outputs.tf
+├── ...
+├── modules/
+│   ├── nestedA/
+│   │   ├── README.md
+│   │   ├── variables.tf
+│   │   ├── main.tf
+│   │   ├── outputs.tf
+│   ├── nestedB/
+│   ├── .../
+├── examples/
+│   ├── exampleA/
+│   │   ├── main.tf
+│   ├── exampleB/
+│   ├── .../
+```
+
```

Comment:
```suggestion
```

Following my other comment I think this should be put it [template/README.md](https://github.com/fredrikhgrelland/vagrant-hashistack/blob/master/template/README.md)